### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by ahmedmaz24**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at mediouniahmed11@yahoo.com. All
```
Comment (rnestler) [inline CODE_OF_CONDUCT.md:58]: I guess you copied the code of conduct from another project? Or where does the email address `mediouniahmed11@yahoo.com` come from?
I think we would need some kind of official way to contact Sensirion with regard to code of conduct violations before we can add it to our repositories.

**Review by rnestler — COMMENTED** [review]
Thanks for your suggestion @ahmedmaz24. Generally I think having a code of conduct is a good thing. But we need to discuss internally how we want to handle code of conducts in general and the enforcement of them especially.
Also note that we don't actively maintain the SmartGadget app anymore.

**Comment by ahmedmaz24** [comment]
Oh this was a premade template made by GitHub and yeah I’m just practicing open source methods for a project 


Sent from Yahoo Mail for iPhone


On Monday, February 25, 2019, 11:13 AM, Raphael Nestler <notifications@github.com> wrote:


@rnestler commented on this pull request.

Thanks for your suggestion @ahmedmaz24. Generally I think having a code of conduct is a good thing. But we need to discuss internally how we want to handle code of conducts in general and the enforcement of them especially.
Also note that we don't actively maintain the SmartGadget app anymore.

In CODE_OF_CONDUCT.md:
> +permanently any contributor for other behaviors that they deem inappropriate,
+threatening, offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at mediouniahmed11@yahoo.com. All

I guess you copied the code of conduct from another project? Or where does the email address mediouniahmed11@yahoo.com come from?
I think we would need some kind of official way to contact Sensirion with regard to code of conduct violations before we can add it to our repositories.

**Comment by rnestler** [comment]
I'll close this for now, since we currently don't have an official way to contact Sensirion with regard to code of conduct violation.

**Pull request closed without merging**